### ISSUE TITLE
AS6104T support, I’ll test

### ISSUE THREAD

**Issue by duhmojo** [mafredri/asustor-platform-driver#29]
Any hope for supporting the as6104t? I’m willing to test. I expect given the hardware it should be fine to run alternative os’ on this box but the trend started a year after it was released. I’m keen on resurrecting mine.

**Comment by DanielGibson** [comment]
Yes, please test it, it actually *might* already work (it's listed as "AS6104T (NOT TESTED!)" in the README), if it doesn't we can hopefully figure it out with your help
